server/services/OpenSearchISMService.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  ILegacyCustomClusterClient,
+  LegacyCallAPIOptions,
+  OpenSearchDashboardsRequest,
+  RequestHandlerContext,
+} from "opensearch-dashboards/server";
+
+export abstract class OpenSearchISMService {
```

Comment:
The class name is a little bit confusing. I propose `MDSEnabledClientService` or something like that sounds more related to the functionality of this service.

Reply:
Agree, I will address this as part of the following commit.